### PR TITLE
Added project viewmodel.

### PR DESCRIPTION
**Pull request by VideoRab**

### CODE REVIEW

File: CV/CV/ViewModel/Project/ProjectViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    public class ProjectViewModel
+    {
+        public int Id { get; set; }
+
+#nullable disable
+        public string Name { get; set; }
+
+        public TimeZoneInfo StartProject { get; set; }
+
+        public TimeZoneInfo EndProject { get; set; }
```
Comment (Blackolf931) [inline CV/CV/ViewModel/Project/ProjectViewModel.cs:12]: Why you use TimeZoneInfo?

Comment (Blackolf931) [inline CV/CV/ViewModel/Project/ProjectViewModel.cs:12]: What about EndProject if the project is under development?

**Review by AngryKirill — APPROVED** [review]

**Review by Blackolf931 — CHANGES_REQUESTED** [review]

File: CV/CV/ViewModel/Project/ProjectViewModel.cs (hunk 1 above)
Comment (VideoRab) [reply CV/CV/ViewModel/Project/ProjectViewModel.cs]: Changed

Comment (VideoRab) [reply CV/CV/ViewModel/Project/ProjectViewModel.cs]: Changed.

**Review by Fanar1n — APPROVED** [review]

**Review by eeveesharp — APPROVED** [review]

**Review by Blackolf931 — APPROVED** [review]

**Pull request merged**